README.md

```diff
@@ -13,7 +13,7 @@ This extension is to help user use [ansible](https://www.ansible.com/) in VSCode
   - Syntax highlighting.
   - Run playbook from Terminal.
     - On Windows, run ansible inside docker.
```

Comment:
If you changed "provide" -> "provides" in the line below, so perhas here "run" -> "runs" as well?
also "ansible" -> Ansible

Reply:
fixed